Description: Parlatype.py should not be executable
 Don't distribute it as a library.
Author: Gabor Karsay <gabor.karsay@gmx.at>

--- a/libreoffice/Makefile.am
+++ b/libreoffice/Makefile.am
@@ -1,6 +1,6 @@
-lodir = $(libdir)/libreoffice/share/Scripts/python
+lodir = $(prefix)/lib/libreoffice/share/Scripts/python
 
 if LO
-dist_lo_SCRIPTS = Parlatype.py
+dist_lo_DATA = Parlatype.py
 endif
 
